packages/core/core.ts

```diff
@@ -71,6 +71,17 @@ declare global {
 }
 
 export async function initFuzzing(options: Options) {
+	const isAsterisks = options.includes.indexOf("*");
```

Comment:
There is already some filter cleanup code in `Instrumentor`, so we should probably move this there as well. If we do that, both "cleanups" could directly be done in the constructor and not, as it's currently the case, for every file.

Reply:
missed that